src/main/resources/application.properties

```diff
@@ -11,4 +11,7 @@ spring.jpa.database-platform = org.hibernate.dialect.MySQL8Dialect
 spring.jpa.hibernate.naming.physical-strategy = org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl
 
 # --- SERVER CONFIGURATION ---
-server.port = 8080
+server.port = 8443
+server.ssl.key-store = classpath:${KEYSTORE_PATH}
+server.ssl.key-store-password = ${KEYSTORE_PASSWORD}
+server.ssl.keyStoreType = PKCS12
```

Comment:
Consider adding an environment variable for the key store type, since some people may want to use other key store types.

Reply:
Done